UNL-CSE-AMBASSADORS/VITA
# Automatic Note for Volunteer Cancel Appointment

Now a note will automatically be added to an appointment when a volunteer cancels an appointment. This will now be consistent with what happens when a volunteer reschedules an appointment and consistent with the client-reschedule stuff.

**TESTING**
1. Sign in as preparer
2. Navigate to appt mgmt page
3. Select any appointment, copy the name or ID so you can search it later
4. Reschedule the appointment
5. Refresh the page and select the same appointment
    * There should be an automatic note saying it was rescheduled by preparer
6. Cancel the appointment
7. Refresh the page and select the same appointment
    * There should be an automatic note saying it was cancelled by preparer
